acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ProcessBesuNodeRunner.java

```diff
@@ -125,6 +125,10 @@ public void startNode(final BesuNode node) {
         params.add("--rpc-http-authentication-credentials-file");
         params.add(node.jsonRpcConfiguration().getAuthenticationCredentialsFile());
       }
+      if (node.jsonRpcConfiguration().getAuthenticationPublicKeyFile() != null) {
```

Comment:
What if `node.jsonRpcConfiguration().getAuthenticationPublicKeyFile()` is empty string, would that cause a problem here?

Reply:
It's a file, so can't be an empty string. But if the file doesn't exist then Besu will fail to start and fail with an exception "Authentication RPC public key could not be read".